pp/task/upload_task.go

```diff
@@ -346,7 +369,30 @@ func (u *UploadFileTask) UpdateSliceDestinationsForRetry(newDestinations []*prot
 			}
 		}
 	}
-	u.retryCount++
+}
+
+func (u *UploadFileTask) SetToPause() {
+	u.mutex.Lock()
+	defer u.mutex.Unlock()
+	u.toPause = true
+}
+
+func (u *UploadFileTask) ResetToPause() {
```

Comment:
The name should be either 'Resume' or 'StopPause'.

Reply:
fixed.